src/components/Troubleshoot/TroubleShootStyles.tsx

```diff
@@ -3,67 +3,101 @@ import { createStyles, Theme } from '@material-ui/core/styles';
 export const styles = (theme: Theme) =>
   createStyles({
     background: {
-      backgroundColor: theme.palette.background.default
+      backgroundColor: theme.palette.background.default,
     },
+
     header: {
       display: "flex",
       flexDirection: 'row',
       alignItems: 'center',
       color: theme.palette.accent2.main,
+      fontWeight: 500,
     },
+
     card: {
-      margin: '10%',
+      margin: '5%',
       position: "relative",
       height: '100vh',
       width: '100%',
       bottom: 0,
-
     },
+
+
     button: {
       borderRadius: '18px',
       marginRight: '1%',
       paddingRight: '2%',
       paddingLeft: '2%',
       borderStyle: 'none',
-    },
-    optionContainer: {
+      fontWeight: 700,
+    }, 
+
+    buttonBoxy :{
       display: 'flex',
       flexDirection: 'row',
       flexWrap: 'wrap',
       alignItems: 'stretch',
       width: 400,
+      height: 60,
+      marginRight: 48,
+      marginBottom: 32,
+      borderRadius: 10,
+      boxShadow: '10px 8px 12px 8px rgba(218, 207, 207, 0.6)',
+      backgroundColor: 'white',
+    }, 
+
+
+    button_back: {
```

Comment:
For `button_back` and `button_restart`, since the first 5ish lines (ie. 51-55, and 62-66) are the same as `button`, I think clean up the repeated styling a bit! (in TroubleshootCard.tsx, all the buttons already use `button` alongside another styling class)